EFCore Power Tools Extension Pack: VsixGallery - Deployment Issue?

Hello, 

I'm troubleshooting some issues I've encountered in updating extensions in VS2022 and noticed that version 2.6.1146 updated 2 days ago may be corrupted?

https://www.vsixgallery.com/extension/EFCorePowerToolsExtensionPack.b2aaf4e1-5424-4a30-9228-7fe89f03f816

Clicking on download returns a 404 error and throws an invalid package error in VS2022.

Looks like something is broken on GitHub preventing me from pushing?

```
2025-09-16T13:39:27.8740545Z [36;1m$ErrorActionPreference='Stop'[0m
2025-09-16T13:39:27.8741169Z [36;1m(new-object Net.WebClient).DownloadString("https://raw.github.com/madskristensen/ExtensionScripts/master/AppVeyor/vsix.ps1") | iex[0m
2025-09-16T13:39:27.8741754Z [36;1mVsix-PublishToGallery [0m
2025-09-16T13:39:27.8774336Z shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
2025-09-16T13:39:27.8774646Z env:
2025-09-16T13:39:27.8774797Z   VERSION: 2.6.1153
2025-09-16T13:39:27.8775317Z   DOTNET_ROOT: C:\Program Files\dotnet
2025-09-16T13:39:27.8776041Z ##[endgroup]
2025-09-16T13:39:28.7664835Z [31;1mParentContainsErrorRecordException: [0mD:\a\_temp\6126e776-0fec-402c-a039-6cc1d5b3f8c8.ps1:3[0m
2025-09-16T13:39:28.7665681Z [31;1m[0m[36;1mLine |[0m
2025-09-16T13:39:28.7666354Z [31;1m[0m[36;1m[36;1m   3 | [0m [36;1m(new-object Net.WebClient).DownloadString("https://raw.github.com/mad[0m …[0m
2025-09-16T13:39:28.7667098Z [31;1m[0m[36;1m[36;1m[0m[36;1m[0m[36;1m     | [31;1m ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~[0m
2025-09-16T13:39:28.7667952Z [31;1m[0m[36;1m[36;1m[0m[36;1m[0m[36;1m[31;1m[31;1m[36;1m     | [31;1mException calling "DownloadString" with "1" argument(s): "The remote server returned an error: (503) hostname[0m
2025-09-16T13:39:28.7668729Z [31;1m[0m[36;1m[36;1m[0m[36;1m[0m[36;1m[31;1m[31;1m[36;1m[31;1m[36;1m     | [31;1mdoesn't match against certificate."[0m
2025-09-16T13:39:28.8169336Z ##[error]Process completed with exit code 1.

```

--- a/.github/workflows/vsix.yml
+++ b/.github/workflows/vsix.yml
@@ -137,7 +137,7 @@ jobs:
       if: github.ref == 'refs/heads/master' && github.repository_owner == 'erikej' && (github.event_name == 'push' ||  github.event_name == 'workflow_dispatch')
       run: |
         $ErrorActionPreference='Stop'
-        (new-object Net.WebClient).DownloadString("https://raw.github.com/madskristensen/ExtensionScripts/master/AppVeyor/vsix.ps1") | iex
+        (new-object Net.WebClient).DownloadString("https://raw.githubusercontent.com/madskristensen/ExtensionScripts/master/AppVeyor/vsix.ps1") | iex
         Vsix-PublishToGallery 
       shell: pwsh
       continue-on-error: false

